lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -282,4 +282,37 @@ export class PublicClientApplication extends ClientApplication implements IPubli
             document.removeEventListener("visibilitychange",this.trackPageVisibility);
         });
     }
+
+    /**
+     * Hydrates cache with the tokens and account in the AuthenticationResult object
+     * @param result
+     * @param request - The request object that was used to obtain the AuthenticationResult
+     * @returns
+     */
+    async hydrateCache(
+        result: AuthenticationResult,
+        request: SilentRequest
+    ): Promise<void> {
+        this.logger.verbose("hydrateCache called");
+
+        if(result.account) {
```

Comment:
`account` is mandated in 3.x, however in 2.x it is till `AccountInfo || null` for `AuthenticationResult`. We can throw an error if `account` is accidentally `null` but I do not see the possibility. Let me know if we need to add that.